Client/Application/ClientPreferences.swift

```diff
@@ -78,5 +78,13 @@ extension Preferences {
         /// The toggles states for clear private data screen
         static let clearPrivateDataToggles = Option<[Bool]>(key: "privacy.clear-data-toggles", default: [])
     }
+    final class Review {
+        /// Application Launch Count (how many times the application has been launched)
+        static let launchCount = Option<Int>(key: "review.launch-count", default: 0)
+        /// Review Threshold (the total amount of launches needed for the next review to show up)
+        static let threshold = Option<Int>(key: "review.threshold", default: 14)
```

Comment:
We should have a static constants for all three thresholds, put it in `AppReview.swift`